azure-mgmt-keyvault/src/main/java/com/microsoft/azure/management/keyvault/CheckNameAvailabilityResult.java

```diff
@@ -0,0 +1,16 @@
+package com.microsoft.azure.management.keyvault;
+
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.keyvault.implementation.CheckNameAvailabilityResultInner;
+import com.microsoft.azure.management.resources.fluentcore.model.HasInner;
+
+/**
+ * Wrapper for CheckNameAvailabilityResultInner
+ * @author tifchen

```

Comment:
Can you please remove this comment and change the overall description of the method to something like "The result of checking for the Key Vault name availability."?